### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by raybellwaves** [prefix-dev/rattler-build#1615] [Show and tell]
Once the discussions tab is open (https://github.com/prefix-dev/rattler-build/issues/1150) this can be moved here.

Thank you for this package! It's great and solves my issue. I was using boa to build a conda package but it keep timing out in CI. I recently switch to ratter-build with great success. I want to document somethings I learn along the way to help anyone else trying this.

Switching the boa/recipe.yaml to a rattler build recipe.yaml took me a little bit as I have to look up some expected fields for the spec. My recipe is fairly basic so only have to switch a few things

### source.git_url -> source.path

```yaml
source:
  git_url: ../
```
->
```yaml
source:
  path: ../
```

### test.imports -> tests.python.imports

```yaml
test:
  imports:
    - "mypkg"
```
->
```yaml
tests:
  - python:
      imports:
        - "mypkg"
```

### about.home -> about.homepage

```yaml
about:
  home: "URL"
```
->
```yaml
about:
  homepage: "URL"
```

In CI I switched from:

```yaml
    - boa build boa-base-recipe --output-folder /tmp/conda
    - boa build boa-recipe --output-folder /tmp/conda
    - for package in "/tmp/conda/noarch"/*;
        do curl -u "${ARTIFACTORY_CLOUD_PUBLISH_USERNAME}:${ARTIFACTORY_CLOUD_PUBLISH_PASSWORD}" -T "$package" "$CONDA_ARTIFACTORY_URL";
      done
```

to 

```yaml
    - rattler-build build --recipe rattler-build-base-recipe/recipe.yaml --output-dir /tmp/conda
    - rattler-build build --recipe rattler-build-recipe/recipe.yaml --channel https://XXX.jfrog.io/artifactory/api/conda/conda/dtn --channel https://XXX.jfrog.io/artifactory/api/conda/conda --output-dir /tmp/conda
    - for package in "/tmp/conda/noarch"/*;
        do curl -u "${ARTIFACTORY_CLOUD_PUBLISH_USERNAME}:${ARTIFACTORY_CLOUD_PUBLISH_PASSWORD}" -T "$package" "$CONDA_ARTIFACTORY_URL";
      done
```

Note I have a mypkg-base with no deps and mypkg will full deps.

**Comment by wolfv** [comment]
Hey @raybellwaves – happy to hear that! 

I think the relative git not working is an issue we'd actually like to fix at some point :) But I am glad that you found a workaround.

**Comment by zelosleone** [comment]
Hey @raybellwaves I think you could also use it like this:

```
source:
  git_url: ../
```

->

```
source:
  git: ../
```
